packages/devtools_test/lib/src/wrappers.dart

```diff
@@ -134,7 +134,12 @@ void testWidgetsWithContext(
     } finally {
       // restore previous global values
       for (Type type in oldValues.keys) {
-        setGlobal(type, oldValues[type]);
+        final oldGlobal = oldValues[type];
+        if (oldGlobal != null) {
```

Comment:
nit: using `oldValues.containsKey` would be safer in case it ever matters whether the oldValue was null rather than ommitted.

Reply:
oldValues[type] will be explicitly set to null above on line 128, so oldValues will actually contain the key here